### PR TITLE
Change pmm-update to ssm-update

### PR DESCRIPTION
**Pull request by oblitorum**

### CODE REVIEW

File: bin/ssm-update-check [hunk 1]
```diff
@@ -86,10 +86,8 @@ main() {
     local PLAYBOOK=$(get_latest_version $RPM_ROOT)/main.yml
     diff $PLAYBOOK $UPDATE_DIR/main.yml
 
-    yum -y update percona-release
-    rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY
-    yum -y --enablerepo=pmm clean all || :
-    yum -y --disablerepo='*' --enablerepo=pmm install pmm-update && EC=$? || EC=$?
+    yum -y --enablerepo=ssm clean all || :
```
Comment (gordan-bobic) [inline bin/ssm-update-check:91]: This should probably include our packaging key.

**Review by gordan-bobic — CHANGES_REQUESTED** [review]

File: bin/ssm-update-check (hunk 1 above)
Comment (oblitorum) [reply bin/ssm-update-check:91]: The `ssm` repo is setup while deploying the docker container by ansible, and the packing key is already defined in repo config. So it doesn't need to import the key again, do I understand this right?

CleanShot 2022-06-20 at 16 31 49@2x

**Comment by gordan-bobic** [comment]
I am not 100% sure if this is going to be a problem, but if you don't explicitly import the key with `rpm --import`, yum will ask to import it. If something is expected to run unattended non-interactive updates, the fact that the key isn't imported explicitly up front could cause a problem.

**Comment by oblitorum** [comment]
> I am not 100% sure if this is going to be a problem, but if you don't explicitly import the key with `rpm --import`, yum will ask to import it. If something is expected to run unattended non-interactive updates, the fact that the key isn't imported explicitly up front could cause a problem.

OK, then I better import the packaging key here.

**Pull request merged**